evalml/pipelines/classification_pipeline.py

```diff
@@ -66,7 +68,14 @@ def fit(self, X, y):
             )
 
         self._fit(X, y)
-        self._classes_ = list(ww.init_series(np.unique(y)))
+
+        # TODO: Added this in because numpy's unique() does not support pandas.NA
+        try:
+            self._classes_ = list(ww.init_series(np.unique(y)))
+        except TypeError as e:
+            if "boolean value of NA is ambiguous" in str(e):
+                self._classes_ = y.unique()
+
```

Comment:
I think this was added just to replace `np.unique` in cases of `pd.NA`, otherwise the behaviour should be the same. @chukarsten was that the intention?